src/services/accounts/AccountResolver.js

```diff
@@ -4,14 +4,23 @@ import { Singleton } from "src/context/steriotypes/Singleton";
 @Singleton
 export class AccountResolver {
   async getAccountNameById(id) {
-    let userPage = await fetch(`https://twitter.com/intent/user?user_id=${id}`);
+    let userPage = await fetch(
+      `https://twitter.com/intent/user?user_id=${id}`,
+      {
+        credentials: "omit"
+      }
+    );
+
     userPage = await userPage.text();
     const userName = this.extractNameFromPage(userPage);
     return userName;
   }
 
   async getIdByAccountName(accountName) {
-    let userPage = await fetch(`https://twitter.com/${accountName}`);
+    let userPage = await fetch(`https://twitter.com/${accountName}`, {
```

Comment:
Here we need to add check for status code to be equal to 200, otherwise returning null should be correct